qt/python/mantidqt/mantidqt/widgets/sliceviewer/peaksviewer/view.py

```diff
@@ -57,6 +57,9 @@ def keyPressEvent(self, event):
         QTableView.keyPressEvent(self, event)
         self._key_handler._row_selected()
 
+    def set_concise(self, bl):
```

Comment:
Might be mistaken but this function and the attribute `is_concise` don't appear to be used?

Reply:
Very well caught thank you